src/metabase/pulse/render/body.clj

```diff
@@ -667,66 +630,100 @@
           card-type     (or (series-setting viz-settings y-col-key :display)
                             chart-type
                             (nth default-combo-chart-types idx))
           selected-rows (mapv #(vector (ffirst %) (nth (second %) idx)) joined-rows)
           y-axis-pos    (or (series-setting viz-settings y-col-key :axis)
                             (nth (default-y-pos data axis-group-threshold) idx))]
-      {:name          card-name
-       :color         card-color
+      {:name          metric-name
+       :cardName      card-name
        :type          card-type
        :data          selected-rows
-       :yAxisPosition y-axis-pos})))
+       :yAxisPosition y-axis-pos
+       :column        y-col})))
 
 (defn- double-x-axis-combo-series
   "This munges rows and columns into series in the format that we want for combo staticviz for literal combo displaytype,
   for a double x-axis, which has pretty materially different semantics for that second dimension, with single y-axis only.
 
   This mimics default behavior in JS viz, which is to group by the second dimension and make every group-by-value a series.
   This can have really high cardinality of series but the JS viz will complain about more than 100 already"
-  [chart-type joined-rows _x-cols _y-cols {:keys [viz-settings] :as data}]
+  [chart-type joined-rows x-cols _y-cols {:keys [viz-settings] :as data} card-name]
   (let [grouped-rows (group-by #(second (first %)) joined-rows)
         groups       (keys grouped-rows)]
     (for [[idx group-key] (map-indexed vector groups)]
       (let [row-group          (get grouped-rows group-key)
             selected-row-group (mapv #(vector (ffirst %) (first (second %))) row-group)
-            card-name          (or (series-setting viz-settings group-key :name)
-                                   (series-setting viz-settings group-key :title)
-                                   group-key)
-            card-color         (or (series-setting viz-settings group-key :color)
-                                   (nth colors idx))
+            column-name          (or (series-setting viz-settings group-key :name)
+                                   (series-setting viz-settings group-key :title))
             card-type          (or (series-setting viz-settings group-key :display)
                                    chart-type
                                    (nth default-combo-chart-types idx))
             y-axis-pos         (or (series-setting viz-settings group-key :axis)
                                    (nth (default-y-pos data axis-group-threshold) idx))]
-        {:name          card-name
-         :color         card-color
+        {:name          column-name
+         :cardName      card-name
          :type          card-type
          :data          selected-row-group
-         :yAxisPosition y-axis-pos}))))
+         :yAxisPosition y-axis-pos
+         :column        (second x-cols)
```

Comment:
For charts with breakout values, FE will set the series key (`String(single.card.name)`) for calculating colors using the each breakout value with this logic
- https://github.com/metabase/metabase/blob/d9a5708306f464e5d512c5d2b7cd18677a1f7667/frontend/src/metabase/visualizations/lib/settings/series.js#L12
- https://github.com/metabase/metabase/blob/d9a5708306f464e5d512c5d2b7cd18677a1f7667/frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx#L482

See the example chart below which explains why `:column` is needed here.

![image](https://user-images.githubusercontent.com/1937582/199700608-303a3095-e676-488f-870e-c206f6527ffd.png)

FE will receive the value like `2016-01-01T00:00:00Z` and pass that along with `column` to `formatValue` and produce the value `2016`. Since this column's type is date, the value is formatted by moment.js.

This is why we need to pass `column` from BE so FE static viz could produce the same series key FE viz does results in the same colors, since these series key are then going to go through some logic which calculate all series colors based on the overall series keys.
https://github.com/metabase/metabase/blob/d9a5708306f464e5d512c5d2b7cd18677a1f7667/frontend/src/metabase/visualizations/lib/settings/series.js#L193